pages.fa/common/calendar.md

```diff
@@ -0,0 +1,20 @@
+# calendar
+
+> نمایش رویداد های پیش رو با استفاده از یک فایل تقویم.
+> اطلاعات بیشتر: <https://www.commandlinux.com/man-page/man1/calendar.1.html>.
+
+- نمایش رویداد های امروز و فردا (یا آخر هفته یعنی روز جمعه) با استفاده از تقویم پیشفرض:
+
+`calendar`
+
+- نگاهی به آینده، نمایش رویداد های 30 روز آینده:
+
+`calendar -A {{30}}`
+
+- نگاهی به گذشته، نمایش رویداد های هفت روز گذشته:
+
+`calendar -B {{7}}`
+
+- نمایش رویداد ها با استفاده از یک فایل تقویم دلخواه:
+
+`calendar -f {{path/to/file}}`
```

Comment:
```suggestion
`calendar -f {{مسیر/به/فایل}}`
```

Can we translate `path/to/file` into farsi? Does this make sense in your opinion? (Also, for catimg.md).